user/common-tasks/mount-from-other-os.md

```diff
@@ -0,0 +1,97 @@
+		│   ├─qubes_dom0-vm--fedora--30--dvm--private--1576749131--back 253:7    0     2G  0 lvm
+		├─qubes_dom0-pool00_tdata                                       253:2    0  84.4G  0 lvm
+		│ └─qubes_dom0-pool00-tpool                                     253:3    0  84.4G  0 lvm
+		│   ├─qubes_dom0-root                                           253:4    0  84.4G  0 lvm   /
+		│   ├─qubes_dom0-pool00                                         253:6    0  84.4G  0 lvm
+		│   ├─qubes_dom0-vm--fedora--30--dvm--private--1576749131--back 253:7    0     2G  0 lvm
+		└─qubes_dom0-swap                                               253:5    0     4G  0 lvm   [SWAP]
+	sdb                                                                   8:16   0 447.1G  0 disk
+	├─sdb1                                                                8:17   0   549M  0 part
+	└─sdb2                                                                8:18   0 446.6G  0 part
+	sr0                                                                  11:0    1  1024M  0 rom
+	nvme0n1                                                             259:0    0 465.8G  0 disk
+	├─nvme0n1p1                                                         259:1    0     1G  0 part
+	└─nvme0n1p2                                                         259:2    0 464.8G  0 part
+
+	```
+2. Decrypt the disk using the command `cryptsetup luksOpen /dev/<disk>`.
+
+Accessing LVM Logical Volumes
+-----------------------------
+
+3. If using an AppVM or standard Linux, LVM should automatically discover the Qubes LVM configuration. In this case, continue to step 4.
+	1. Qubes uses the default name `qubes_dom0` for it's LVM LV.
+	   This will conflict with the name of the LV of the currently installed system.
+	   To read both, you will have to rename the VG.
+	   *Note:* If this is not reversed, the Qubes install being accessed will not be bootable.
+	2. Find the UUID of the LV to be accessed using the command `lvdisplay`.
+	   This will be the LV named `qubes_dom0` which is not marked active.
+	3. The command `lvrename <UUID> other_install` will rename the LV.
+4. Run the command `lvscan` to add any new LVs to the device list.
+
+Mounting the disk
+-----------------
+
+5. Find the disk to be accessed. The `lsblk` command above may be of use. The following rules apply by default:
+
+| Disk name           			| Data type     	| Explination   			    			  |
+| ----------------------------- | ----------------- | ------------------------------------------- |
+| other\_install-root     		| dom0 root 		| The root partition of dom0. 			 	  |
+| other\_install-<vm>-private   | VM  				| The /rw partition of the named VM. 		  |
+| other\_install-<vm>-root      | templateVM root 	| The root partition of the named TemplateVM. |
+| other\_install\_tmeta     	| LVM Metadata 		| The metadata LV of this disk. 			  |
+
+6. Mount the disk using the command `mount /dev/mapper/other_install-<lv name> <mountpoint>`.
```

Comment:
Names in `/dev/mapper` are mangled (`-` is replaced with `--`). It may be more convenient to use symlinks in `/dev/other_install/`